dspace-server-webapp/src/test/java/org/dspace/app/rest/BitstreamRestControllerIT.java

```diff
@@ -51,42 +70,112 @@
 import org.dspace.statistics.SolrLoggerServiceImpl;
 import org.dspace.statistics.factory.StatisticsServiceFactory;
 import org.dspace.statistics.service.SolrLoggerService;
+import org.junit.After;
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 
 /**
- * Integration test to test the /api/core/bitstreams/[id]/content endpoint
+ * Integration test to test the /api/core/bitstreams/[id]/* endpoints
  *
  * @author Tom Desair (tom dot desair at atmire dot com)
  * @author Frederic Van Reet (frederic dot vanreet at atmire dot com)
  */
-public class BitstreamContentRestControllerIT extends AbstractControllerIntegrationTest {
+public class BitstreamRestControllerIT extends AbstractControllerIntegrationTest {
 
     protected SolrLoggerService solrLoggerService = StatisticsServiceFactory.getInstance().getSolrLoggerService();
 
-    private static final Logger log = LogManager
-        .getLogger(BitstreamContentRestControllerIT.class);
-
     @Autowired
     private ConfigurationService configurationService;
 
     @Autowired
     private CitationDocumentServiceImpl citationDocumentService;
 
+    @Autowired
+    private BitstreamService bitstreamService;
+
+    @Autowired
+    private BitstreamFormatService bitstreamFormatService;
+
+    @Autowired
+    private ResourcePolicyService resourcePolicyService;
+
+    private Bitstream bitstream;
+    private BitstreamFormat supportedFormat;
+    private BitstreamFormat knownFormat;
+    private BitstreamFormat unknownFormat;
+    private ResourcePolicy resourcePolicy;
+
     @BeforeClass
     public static void clearStatistics() throws Exception {
         // To ensure these tests start "fresh", clear out any existing statistics data.
         // NOTE: this is committed immediately in removeIndex()
-        StatisticsServiceFactory.getInstance().getSolrLoggerService().removeIndex("*:*");
+//        StatisticsServiceFactory.getInstance().getSolrLoggerService().removeIndex("*:*");
```

Comment:
This should not be commented out, as it leaves the `clearStatistics()` method empty.  Is there a reason this needs to be commented out?

Reply:
sorry, was a mistake!